src/user.rs

```diff
@@ -277,25 +279,39 @@ impl User<LoggedIn> {
             }
         });
 
+        let mut rooms = vec![];
+
+        let response = client.sync_once(SyncSettings::default()).await;
+
+        if let Ok(res) = response {
+            for (room, _) in res.rooms.join {
+                rooms.push(room);
+            }
+        }
+
         User {
             id: self.id.clone(),
             client: self.client.clone(),
             tx: self.tx.clone(),
             state: Synching {
-                rooms: Arc::new(Mutex::new(vec![])),
+                rooms: Arc::new(Mutex::new(rooms)),
+                available_room_ids: vec![],
             },
             friendships: self.friendships.clone(),
         }
     }
 }
 
 impl User<Synching> {
-    pub async fn create_room(&mut self) -> Option<Box<RoomId>> {
+    pub async fn create_room(&mut self, friendship: &Friendship) -> Option<Box<RoomId>> {
         let client = self.client.lock().await;
 
         let instant = Instant::now();
-        let request = CreateRoomRequest::new();
+        let mut request = CreateRoomRequest::new();
+        let alias = friendship.local_part.to_string();
+        request.room_alias_name = Some(&alias);
```

Comment:
nit: Change in favor of `assign!` macro and preventing from using mutable vars.
```suggestion
        let request = assign!(CreateRoomRequest::new(), { room_alias_name: Some(&alias) });
```